### DISCUSSION TITLE
How to join a slack channel?

### DISCUSSION THREAD

**Discussion by mango1135** [arrow-kt/arrow#3509] [Q&A] [answered]
I tried to join a Slack channel, but it seems like you can only join with a JetBrains email.

Is there any way to join with a Gmail account on Slack?

**Comment by serras** [comment, marked as answer]
In theory you can just go to https://slack-chats.kotlinlang.org/ and press the _Join Slack_ button in the top-right corner. You should be able to join with any e-mail you like.
